python/python/glide/async_commands/core.py

```diff
@@ -1713,16 +1713,64 @@ async def zrem(
             If `key` does not exist, it is treated as an empty sorted set, and the command returns 0.
 
         Examples:
-            >>> await zrem("my_sorted_set", ["member1", "member2"])
+            >>> await client.zrem("my_sorted_set", ["member1", "member2"])
                 2  # Indicates that two members have been removed from the sorted set "my_sorted_set."
-            >>> await zrem("non_existing_sorted_set", ["member1", "member2"])
+            >>> await client.zrem("non_existing_sorted_set", ["member1", "member2"])
                 0  # Indicates that no members were removed as the sorted set "non_existing_sorted_set" does not exist.
         """
         return cast(
             int,
             await self._execute_command(RequestType.Zrem, [key] + members),
         )
 
+    async def zremrangebyscore(
+        self,
+        key: str,
+        min_score: Union[InfBound, ScoreBoundary],
+        max_score: Union[InfBound, ScoreBoundary],
```

Comment:
```suggestion
        min_score: Union[InfBound, ScoreBoundary, LexBoundary],
        max_score: Union[InfBound, ScoreBoundary, LexBoundary],
```

Update doc, example and implementation as well

Reply:
Oh, that is my mistake. I messed `zrange` and `zremrange` - they have completely different syntax. Please, ignore my comments about `LexBoundary`